### PR TITLE
[NavigatorIOS] added shadowHidden feature

### PR DESCRIPTION
**Pull request by brentkirkland**
PR #1615 already addressed this issue but did not include implementation details in the RCTWrapperViewController class @implementation block. This provides better insight into objective-c files.

### CODE REVIEW

File: React/Views/RCTWrapperViewController.m [hunk 1]
```diff
@@ -133,4 +140,18 @@ - (void)didMoveToParentViewController:(UIViewController *)parent
   }
 }
 
+- (UIImageView *)findHairlineImageViewUnder:(UIView *)view {
```
Comment (sahrens) [inline React/Views/RCTWrapperViewController.m]: Formatting is off.  Make sure to use 2 space indents and curly braces should be on their own line.

File: React/Views/RCTWrapperViewController.m [hunk 2]
```diff
@@ -133,4 +140,18 @@ - (void)didMoveToParentViewController:(UIViewController *)parent
...
+    if ([view isKindOfClass:UIImageView.class] && view.bounds.size.height <= 1.0) {
```
Comment (sahrens) [inline React/Views/RCTWrapperViewController.m]: This isn't the worst thing in the world, but there isn't a better way to grab the shadow?

**Comment by sahrens** [comment]
Closing in favor of the earlier #1615.

File: React/Views/RCTWrapperViewController.m (hunk 2 above)
Comment (brentkirkland) [inline React/Views/RCTWrapperViewController.m]: Unfortunately no, this way preserves transparency. Other easier ways quickly mess up the transparency issue.

**Pull request closed without merging**